### PR TITLE
Added GitHub Actions for CI and updated README

### LINKED ISSUE #41: [Feature] Use GitHub Actions for CI
It would be great to use GitHub Actions for CI instead of travis. To resolve this issue, complete the following:
- replace the travis.yml file with a github action workflow that runs linting, tests, and the build. I think it would be just like the github action recently added to react-paypal-js: https://github.com/paypal/react-paypal-js/pull/33
- replace the travis ci badge in the readme with a github actions badge

### PR DESCRIPTION
**Pull request by logesr**
Resolves #41

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```
Comment (gregjopa) [inline .github/workflows/main.yml]: Please update to support both Node 10 and 12 like this: https://github.com/paypal/react-paypal-js/blob/main/.github/workflows/main.yml#L11.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        run: npm install
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Test
+        run: npm run test
+
+      - name: Build
+        run: npm run build
```
Comment (gregjopa) [inline .github/workflows/main.yml]: Please add a blank line at the end of the file.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ A client-side loader for the [PayPal JS SDK](https://developer.paypal.com/docs/b
 
 <a href="https://www.npmjs.com/package/@paypal/paypal-js"><img src="https://img.shields.io/npm/v/@paypal/paypal-js?style=flat-square" alt="npm version"></a>
 <a href="https://github.com/paypal/paypal-js/blob/main/LICENSE.txt"><img src="https://img.shields.io/npm/l/@paypal/paypal-js?style=flat-square" alt="github license"></a>
-<a href="https://travis-ci.com/github/paypal/paypal-js"><img src="https://img.shields.io/travis/com/paypal/paypal-js/main.svg?style=flat-square" alt="travis build"></a>
+![CI](https://github.com/paypal/paypal-js/workflows/CI/badge.svg)
```
Comment (gregjopa) [inline README.md]:
```suggestion
<a href="https://github.com/paypal/paypal-js/actions?query=workflow%3ACI"><img src="https://github.com/paypal/paypal-js/workflows/CI/badge.svg" alt="CI Status"></a>
```

**Comment by gregjopa** [comment]
Thanks for the PR @logesr! I left a few change requests.

**Comment by logesr** [comment]
@gregjopa I have made the changes in the latest commit

**Pull request merged**